int64 arm
---
bool res
